apps/site/assets/css/_header.scss

```diff
@@ -567,16 +567,16 @@ nav.m-menu--mobile {
   margin: auto 0;
 
   &:hover,
-  &:focus {
+  &:focus-visible {
     background-color: $brand-primary-darkest;
   }
 
-  &:focus {
-    border-color: $brand-primary-light;
```

Comment:
Can we add these back in? This removed the focus styling applied when tabbing through the desktop navigation buttons (it used to look like below). Maybe adjusting these to `:focus:not(:focus-visible)` might get what we want for the border, or maybe we might need to style the desktop menu buttons and mobile menu buttons differently 🤔 

![image](https://user-images.githubusercontent.com/2136286/154296277-116121e2-2bb5-4a96-9bed-96a55c2049ad.png)

Reply:
Just looked into this, it appears that the focus styling is applying when tabbing through these with a keyboard on my end. Unless you're referring to the fact that the outline is now `$brand-secondary` which I thought was desired according to the mocks I saw. 

I could also be totally misunderstanding what you're pointing out so let me know :)

Reply:
Hm interesting!
Is this on dev-blue? On that the button background is transparent/idle blue when I tab to it -- this is only on Safari 15.3, it looks fine in FF... 
<img width="915" alt="image" src="https://user-images.githubusercontent.com/2136286/154357482-7feb9dcc-6488-4fd8-b8a1-f2ccc5520ee9.png">

Reply:
This is on dev-blue yes (you should see a network request for `focus-visible.js`). 

I see what's going on, one sec

Reply:
Should be good now (was missing the polyfill selector in the stylesheet)

Reply:
Okay, the script tag was being built and `focus-visible.js` was being requested but Safari wasn't loading it for me. I ran it locally and finally got it to work in my Safari by `require`ing it conditionally -- let me know what you think!